webapp/src/main/java/com/box/l10n/mojito/service/security/user/UserService.java

```diff
@@ -437,7 +437,8 @@ public User getServiceAccountUser(String serviceName) {
       logger.error(
           "Service '{}' attempted and failed authentication. No matching services found.",
           serviceName);
-      return null;
```

Comment:
Does this have any impact outside of the auth flow?

Reply:
It does not. This is a new method that I added a few weeks ago. it is strictly used in the Auth flow